tests/meltano/cli/test_schedule.py

```diff
@@ -37,22 +39,165 @@ def test_schedule(self, get, session, project, cli_runner, schedule_service):
         assert_cli_runner(res)
         schedule = schedule_service.schedules()[0]
 
-        assert schedule.name == "schedule-mock"
+        assert schedule.name == "elt-schedule-mock"
         assert schedule.extractor == "tap-mock"
         assert schedule.loader == "target-mock"
         assert schedule.transform == "run"
         assert schedule.interval == "@eon"  # not anytime soon ;)
-        assert schedule.start_date == iso8601_datetime(TEST_DATE)
+        assert schedule.start_date == iso8601_datetime(test_date)
+
+        # test adding a scheduled job
+        with mock.patch(
+            "meltano.cli.schedule.ScheduleService", return_value=schedule_service
+        ):
+            res = cli_runner.invoke(
+                cli,
+                [
+                    "schedule",
+                    "job-schedule-mock",
+                    "--job",
+                    "mock-job",
+                    "--interval",
+                    "@eon",
+                ],
+            )
+        assert res.exit_code == 0
+
+        assert_cli_runner(res)
+        schedule = schedule_service.schedules()[1]
+
+        assert schedule.name == "job-schedule-mock"
+        assert schedule.job == "mock-job"
+        assert schedule.interval == "@eon"  # not anytime soon ;)
+
+        # verify that you can't use job and elt flags together
+        with mock.patch(
+            "meltano.cli.schedule.ScheduleService", return_value=schedule_service
+        ):
+            res = cli_runner.invoke(
+                cli,
+                [
+                    "schedule",
+                    "job-schedule-mock",
```

Comment:
Should this be "schedule", "add",...

```suggestion
                    "schedule",
                    "add",
                    "job-schedule-mock",
```

Or if we are supporting both methods, perhaps the test should test both ways?